### PR TITLE
added Turkish Language

### PR DESCRIPTION
**Pull request by volkankoc**
added Turkish Language

### CODE REVIEW

**Comment by volkankoc** [comment]
please add turkish language

File: src/i18n/locales/tr.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
+const english = {
```
Comment (1ilit) [inline src/i18n/locales/tr.js:1]: Please rename this to 'turkish'

**Review by 1ilit — CHANGES_REQUESTED** [review]

File: src/i18n/locales/tr.js (hunk 1 above)
Comment (volkankoc) [reply src/i18n/locales/tr.js]: i change it

**Review by volkankoc — COMMENTED** [review]
I change it

**Review by 1ilit — APPROVED** [review]

**Pull request merged**